### ISSUE TITLE
Certificate for NA Endpoint error

### ISSUE THREAD

**Issue by naseemr** [programmerben/KodiKontrol#3]
I am copying the following per your instruction and getting error" Invalid ssl cerificate

-----BEGIN CERTIFICATE-----
MIIDdDCCAlygAwIBAgIJAPp3fm8LcyrFMA0GCSqGSIb3DQEBCwUAMFgxCzAJBgNV
BAYTAlVTMQswCQYDVQQIEwJDQTEVMBMGA1UEBxMMU2FudGEgTW9uaWNhMRYwFAYD
VQQKEw1TbWFydEhvbWVTb2Z0MQ0wCwYDVQQDEwRLb2RpMB4XDTE2MDEzMTA3NDUz
N1oXDTE3MDEzMDA3NDUzN1owWDELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRUw
EwYDVQQHEwxTYW50YSBNb25pY2ExFjAUBgNVBAoTDVNtYXJ0SG9tZVNvZnQxDTAL
BgNVBAMTBEtvZGkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCkctQe
SxQEab/8hVzMLi5uEDn9FbxKja8PMbqPH6V7nai2z2FnOOcR/khd7qnHxDe/JWXu
PcjwlNO0rjZYKynxZHzJFiqUNccTy+6ynbeNCQp03N4Exw2FMVi2LOLR4rlftXGl
lCvLIbbJWPTFcDEcZXr/7n4gHqZe4+52Erg8+dBPzV2KuXZpOcQd6LmdjrN4BZHR
/MqQfQ2uDwQUashwQeWlXZzeCSCYCWlw1rJQ+JPgQVBwZbFbw8eoY86FSZcl2sUM
jrtEQQAzOa1BW6Oa61MdZhcPZL1SEis8lwD/lur8dzRnv2v+Og0jLi+sDQHMYKSq
9X/3ZHzGQ8wlU7o3AgMBAAGjQTA/MAsGA1UdDwQEAwIEMDATBgNVHSUEDDAKBggr
BgEFBQcDATAbBgNVHREEFDASghB3d3cudHJvbGxib3Qub3JnMA0GCSqGSIb3DQEB
CwUAA4IBAQBEYMZNH+0zp/csifIK5+EoMKxnALDOpZQtmJnFjkknutEtPcoElJ8q
DeMb2VebyORmd9bX7RQka4hUA5d38pZk+lbjMBmR/wdLOcfpWR+ielgsYDpimbTG
Njho6I5RC68ReWKySppS6RwH/OXbL6JuHp2vmiM32nPYM0TUs6CqpbXE1cQhmJLr
TMptFFlAnTkjYixdH9suaaAHggiXbmn/sWKtHxytCjJD6cA0cwK4GHO+ETwfGAWj
oDjabINYzBxeFxTNGb4CT27hnqy00VKLZmNZAsOet6Lrpf8F1v7in/niKl/nWqTQ
nNPqjzMWPsvL4wwBpgvLapCn7XtzlXAw
+-----END CERTIFICATE-----

**Comment by benburkhart1** [comment]
Yeah you should really generate your own certificate with letsencrypt.org, it's free. If you can wait until this weekend I will be submitting a new release of KodiKontrol with more accurate directions.

**Comment by naseemr** [comment]
finally got that resolved. No when I start the server, I get error "Port 443 is already in use"

**Comment by benburkhart1** [comment]
Sounds like you have something already using port 443. Do you have apache or nginx listening on those ports? It's the standard port for SSL

**Comment by naseemr** [comment]
I have Apache but it is configured for port 80. Any chance I can use a different port than 443?

**Comment by benburkhart1** [comment]
I don't think amazon will allow it, but you can try. Try killing apache and seeing if that error persists.
